pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -318,8 +318,20 @@ func (c *Cluster) configureMgrModules() error {
 
 		if module.Enabled {
 			if module.Name == balancerModuleName {
+				force := false
+				// Get current "client" features
+				features, err := client.CheckFeatures(c.context, c.Namespace)
```

Comment:
Why do we need to check the features? Could we just always force it? If the desired state is to enable the balancer module, that seems acceptable to force it.

Reply:
@jdurgin, thoughts?

Reply:
I guess what could be useful is at least to display that we are forcing it because old clients were detected. wdyt?

Reply:
What are the side effects? Those clients won't be able to connect to the cluster? The operator log may not be somewhere they would look to troubleshoot that. I'd vote to keep the code simple and not detect it.

Reply:
Yes probably better to just force it.

Reply:
Done;